src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -100,6 +100,16 @@ protected ActionResult execute() {
             data.emailToEdit.emailId = emailId;
         }
 
+        if (addressModeOn && groupModeOn) {
+            statusToAdmin = "Email queued for sending." + "<br/>" + "Receipient: " + addressReceiverListString
```

Comment:
@xpdavid I did think of doing this way, but then statusToAdmin will have "Email queued: " when group mode and address mode both are not on, as said by @Neurrone in the previous review, this may create confusion for admin as the log will only have "Email queued:" without any recipient.